config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/loader/StoredPasswordsLoader.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.tasks.zip.loader;
+
+import com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.Bundle;
+import com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.StoredPassword;
+import com.ca.apim.gateway.cagatewayconfig.util.file.FileUtils;
+
+import java.util.Map;
+
+import static java.util.function.Function.identity;
+import static java.util.stream.Collectors.toMap;
+import static org.apache.commons.lang.StringUtils.isEmpty;
+
+public class StoredPasswordsLoader extends PropertiesLoaderBase {
+
+    private static final String STORED_PASSWORDS_PROPERTIES = "config/stored-passwords.properties";
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: 'PASSWORD' detected in this expression, review this potentially hard-coded credential. ([squid:S2068](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS2068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=70&issues=AWXpQpBKOlZ5QoPzkFI-&open=AWXpQpBKOlZ5QoPzkFI-)

[ISSUE_KEY]: AWXpQpBKOlZ5QoPzkFI-